services/api/src/resources/problem/resolvers.ts

```diff
@@ -9,17 +9,37 @@ const logger = require('../../logger');
 export const getAllProblems: ResolverFn = async (
   root,
   args,
-  { sqlClient, hasPermission }
+  {
+    sqlClient,
+    hasPermission,
+    models,
+    keycloakGrant
+  },
 ) => {
   let rows = [];
+  let userProjectIds = [];
 
   try {
-    if (!R.isEmpty(args)) {
-      rows = await problemHelpers(sqlClient).getAllProblems(args.source, args.environment, args.envType, args.severity);
-    }
-    else {
-      rows = await query(sqlClient, Sql.selectAllProblems({source: [], environmentId: 0, environmentType: [], severity: []}));
+    await hasPermission('problem', 'viewAll');
```

Comment:
We can remove this permission and this entire try block's logic (after our discussion)